src/community/json-ld/src/main/java/org/geoserver/jsonld/request/JsonLdTemplateCallback.java

```diff
@@ -40,33 +41,28 @@ public JsonLdTemplateCallback(GeoServer gs, JsonLdConfiguration configuration) {
 
     @Override
     public Operation operationDispatched(Request request, Operation operation) {
-        FeatureTypeInfo typeInfo = null;
-        GetFeatureRequest getFeature = GetFeatureRequest.adapt(operation.getParameters()[0]);
-
-        if (getFeature != null) {
-            Map rawKvp = request.getRawKvp();
-            boolean jump =
-                    request.getHttpRequest().getMethod().equalsIgnoreCase("GET")
-                            && (rawKvp.get("CQL_FILTER") == null
-                                    || !rawKvp.get("CQL_FILTER").toString().contains("."));
-            if (!jump) {
-                for (int i = 0; i < getFeature.getQueries().size(); i++) {
-                    Query q = getFeature.getQueries().get(i);
+        if (!request.getHttpRequest().getServletPath().equals("/ogc")
```

Comment:
Servlet path tests are fragile:

- The service could be a workspace specific one, not 100% sure to do the various wrappers, but at this point you might see "workspace/ogc" as the servlet path
- The OGC services might end up being moved to another position before upgrade (not sure)
- Other plugins in the future might be using json-ld (e.g., dunno, GetFeatureInfo, a WPS process) but not based on this plugin

It seems safer to me, to use ``"WFS".equalsIgnoreCase(request.getService())`` to test for WFS, along with the output format type.

Reply:
fixed